### ISSUE TITLE
Sonarr LXC (maybe others as well) is ignoring the advanced settings for unprivileged = False

### ISSUE THREAD

**Issue by Dryadwoods** [tteck/Proxmox#1279]
I have tried to install the Sonarr LXC with different advanced settings and no matter if I chose unprivileged = false or = true... my value is always ignored and the container is always created with unprivileged = true

With unprivileged = true I was not able to mount a cifs
I manually had to edit the container conf file to remove the unprivileged.
This change allowed me to make the mount... but the sonnar application was non longer working because 
 "This program is not installed setuid root"
 
 So, my fix was to apt remove sonarr ... followed with apt install sonarr...

In conclusion: just fix the advanced installation to use the correct unprivileged value.
Thank you.

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/issues/394#issuecomment-1201338725 spacebar = select
